releases_cmd_test.go

```diff
@@ -69,6 +69,12 @@ func TestReleasesCmdPrintVersin(t *testing.T) {
 			"public": {"index.html": "2", "docs.html": "2"}
 			}`,
 		},
+		{Version: "v2",
```

Comment:
this does not really tests the above change.
both versions have to be "v1" and it only prints the first one.